test/provision_vagrant

```diff
@@ -17,7 +17,7 @@ apt install --yes --force-yes -q build-essential ruby ruby-dev sqlite3 libsqlite
 (
   echo Install chruby
   [ -s chruby-0.3.9.tar.gz ] || wget -q -O chruby-0.3.9.tar.gz https://github.com/postmodern/chruby/archive/v0.3.9.tar.gz
-  [ chruby-0.3.9 ] || tar -xzf chruby-0.3.9.tar.gz
+  [ -s chruby-0.3.9 ] || tar -xzf chruby-0.3.9.tar.gz
```

Comment:
ok @ianheggie, but what about this line? as far as I know, doing `[ chruby-0.3.9 ]` does nothing.

Reply:
I am focusing on pre_rails4, rails2.3 and fix_middleware_insert all of which have the fixed version of the provisioning. fix_middleware_insert is branched off rails51. When I have got the fix_middleware_insert tests working then it will be folded back into master and released and I will clean out the other branches.

Reply:
thanks! 👍